vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -2089,11 +2092,10 @@ private HomeServerConnectionConfig getHsConfig() {
 
             try {
                 mHomeserverConnectionConfig = null;
```

Comment:
Why is there still this line? is it necessary?

Reply:
in case of exception I suppose...